mkosi/resources/mkosi.md

```diff
@@ -1498,20 +1503,30 @@ boolean argument: either `1`, `yes`, or `true` to enable, or `0`, `no`,
 `QemuFirmware=`, `--qemu-firmware=`
 
 : When used with the `qemu` verb, this option specifies which firmware
-  to use. Takes one of `uefi`, `bios`, `linux`, or `auto`. Defaults to
-  `auto`. When set to `uefi`, the OVMF firmware is used. When set to
+  to use. Takes one of `uefi`, `uefi-secure-boot`, `bios`, `linux`, or
+  `auto`. Defaults to `auto`. When set to `uefi`, the OVMF firmware
+  without secure boot support is used. When set to `uefi-secure-boot`,
+  the OVMF firmware with secure boot support is used. When set to
   `bios`, the default SeaBIOS firmware is used. When set to `linux`,
   direct kernel boot is used. See the `QemuKernel=` option for more
   details on which kernel image is used with direct kernel boot. When
-  set to `auto`, `linux` is used if a cpio image is being booted, `uefi`
+  set to `auto`, `uefi-secure-boot` is used if possible and `linux`
   otherwise.
 
 `QemuFirmwareVariables=`, `--qemu-firmware-variables=`
 
 : When used with the `qemu` verb, this option specifies the path to the
   the firmware variables file to use. Currently, this option is only
-  taken into account when the `uefi` firmware is used. If not specified,
-  mkosi will search for the default variables file and use that instead.
+  taken into account when the `uefi` or `uefi-secure-boot` firmware is
+  used. If not specified, mkosi will search for the default variables
+  file and use that instead.
+
+: When set to `microsoft`, a firmware variables file with the Microsoft
+  secure boot certificates already enrolled will be used.
+
+: When set to `custom`, the secure boot certificate from
+  `SecureBootCertificate=` will be enrolled into the default firmware
+  variables file.
 
 : `virt-fw-vars` from the
   [virt-firmware](https://gitlab.com/kraxel/virt-firmware) project can
```

Comment:
Maybe drop the link and simply mention that it is installed in the tools tree.

Also GH does not let me annotate it but I guess the below section about where Debian and Fedora ship ovmf variables with MS keys enrolled is now superfluous?